src/app/components/permits/VacationRental/VacationRental.tsx

```diff
@@ -1,16 +1,16 @@
-import { HolidayRental } from "@amsterdam/wonen-ui"
-import { usePermitDetails } from "app/state/rest"
+import { HolidayRentalReports } from "@amsterdam/wonen-ui"
+import { useMeldingen } from "app/state/rest"
 
 type Props = {
   bagId: string
 }
 
 const VacationRental: React.FC<Props> = ({ bagId }) => {
-  const [data, { isBusy }] = usePermitDetails(bagId)
+  const [data, { isBusy }] = useMeldingen(bagId)
```

Comment:
Dit is een andere destructure van useMeldingen hook dan in je [andere PR](https://github.com/Amsterdam/fixxx-looplijsten-frontend/pull/1937/files#)